client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -70,8 +69,8 @@
     private final   Set<PreRequestHook>         preRequestHooks;
     private         RxClientProvider            clientProvider;
     private         ObjectMapper                objectMapper;
-    private         int                         timeout     = 10;
-    private         TimeUnit                    timeoutUnit = TimeUnit.SECONDS;
+    private         int                         timeout;
+    private         TimeUnit                    timeoutUnit = TimeUnit.MILLISECONDS;
```

Comment:
static final?

Reply:
Sorry can't do that.
We still want to be able to change these settings by calling the setTimeout method